BackendTests/tools/MetAPITools.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System.Collections.ObjectModel;
+
+namespace BackendTests.tools {
```

Comment:
Capitalise `tools` in namespace and folder for consistency. Sider might have complained about this as well.

Reply:
I don't think that should be the case here, and it was a conscious choice. While conforming to the standard is nice, using non-capitalized names for directories solely handling static helper classes let's us be more clear in cases where the helpers are used. 

I'm not sure why this is the case, but if it's capitalized, accessing a helper-variable would look like this:
- "Tools.StaticHelperClass.StaticHelperVariable" instead of 
- "StaticHelperClass.StaticHelperVariable"

In my opinion, the latter is better, as cleaner code beats a one-off in the folder structure. Agree/disagree?

Reply:
Hm, sound arguments! Didn't consider that.
I see your concern, and agree with the points you raise.

Might there be a better name that conveys what the files in the folder do? Tools to me sounds like somewhere you would place external tools that perform a function.

Perhaps `utils` or `helpers` which are commonly used elsewhere?

If you find the name `tools` sufficiently clear, ignore my comment. Might just be because I'm not used to the C# _way of life_.

Reply:
Renaming to utils.